backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -153,16 +154,62 @@ public Person(
     this.testResultDeliveryPreference = testResultDeliveryPreference;
   }
 
-  public Person(PersonName names, Organization org, Facility fac) {
+  public Person(
+      Organization organization,
+      Optional<Facility> facility,
+      String lookupId,
+      String firstName,
+      String middleName,
+      String lastName,
+      String suffix,
+      LocalDate birthDate,
+      StreetAddress address,
+      String country,
+      PersonRole role,
+      List<String> emails,
+      String race,
+      String ethnicity,
+      List<String> tribalAffiliation,
+      String gender,
+      Boolean residentCongregateSetting,
+      Boolean employedInHealthcare,
+      String preferredLanguage,
+      TestResultDeliveryPreference testResultDeliveryPreference) {
+    this(
+        organization,
+        lookupId,
+        firstName,
+        middleName,
+        lastName,
+        suffix,
+        birthDate,
+        address,
+        country,
+        role,
+        emails,
+        race,
+        ethnicity,
+        tribalAffiliation,
+        gender,
+        residentCongregateSetting,
+        employedInHealthcare,
+        preferredLanguage,
+        testResultDeliveryPreference);
+    if (facility.isPresent()) {
+      this.facility = facility.get();
+    }
+  }
+
+  public Person(PersonName names, Organization org, Facility facility) {
     super(org);
-    this.facility = fac;
+    this.facility = facility;
```

Comment:
Thank you!! I don't like the short variable names 😆

Reply:
pet peeve of mine 😂